transport/model/config.go

```diff
@@ -0,0 +1,53 @@
+package model
+
+import (
+	svcmodel "release-manager/service/model"
+)
+
+type UpdateConfigRequest struct {
+	OrganizationName      *string                        `json:"organization_name"`
+	DefaultReleaseMessage *string                        `json:"default_release_message"`
+	Slack                 UpdateIntegrationConfigRequest `json:"slack"`
+	Github                UpdateIntegrationConfigRequest `json:"github"`
+}
+
+type UpdateIntegrationConfigRequest struct {
+	Enabled *bool   `json:"enabled"`
+	Token   *string `json:"token"`
+}
+
+type ConfigResponse struct {
+	OrganizationName      string                    `json:"organization_name"`
+	DefaultReleaseMessage string                    `json:"default_release_message"`
+	Slack                 IntegrationConfigResponse `json:"slack"`
+	Github                IntegrationConfigResponse `json:"github"`
+}
+
+type IntegrationConfigResponse struct {
+	Enabled bool   `json:"enabled"`
+	Token   string `json:"token"`
+}
+
+func ToSvcConfigUpdate(orgName, rlsMsg *string, slack, github UpdateIntegrationConfigRequest) svcmodel.ConfigUpdate {
```

Comment:
Thanks, make sense 👍

One of the first reviews when I was passing the whole struct, Marek suggested to [pass each param explicitly](https://github.com/jan-zabloudil/release-manager/pull/4#discussion_r1517570203). But I probably got the point slightly wrong and therefore used this approach lately more widely than I should.

Reply:
Aha, I see.
Also, good point by @CermakM - sort of mirror side effect of the "whole struct" approach..

But in general, I wouldn't be afraid of it, because you'll have it covered by tests, and if you miss something, you will notice and easily add it there.

It's fair to keep in mind though that we do not have end-to-end tests yet (as we discussed on Slack).
But still, I presume that it will be covered at least by some manual or semi-automatic testing at some point and will be discovered...

Also, looking at it a bit more - verbosity and semantics. This method does a trivial thing and should look trivial.
But when it has so many input arguments, it sort of hints that this method does something huge and that each argument plays an important role, when in reality it doesn't.